cmake/GzInstallAllHeaders.cmake

```diff
@@ -157,31 +157,36 @@ function(gz_install_all_headers)
 
     set(component_name ${gz_install_all_headers_COMPONENT})
 
-    # Define the install directory for the component meta header
-    set(meta_header_install_dir ${GZ_INCLUDE_INSTALL_DIR_FULL}/${PROJECT_INCLUDE_DIR}/${component_name})
+    # Define the install directory for the "config" header
+    # The "meta" header will be installed one folder above this
+    set(config_header_install_dir ${GZ_INCLUDE_INSTALL_DIR_FULL}/${PROJECT_INCLUDE_DIR})
```

Comment:
I messed up on this line, will fix

Reply:
https://github.com/gazebosim/gz-cmake/pull/467/commits/bb9f35f66d43e238bb9b934d4292dcb2135e9020